### ISSUE TITLE
False positive on "Cannot move" inspection on a type with derived Copy

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#4850]
## Environment

* **IntelliJ Rust plugin version:**
* **Rust toolchain version:**
* **IDE name and version:**
* **Operating system:**

## Problem description

From https://github.com/intellij-rust/intellij-rust/issues/3101#issuecomment-576413621:

> I just noticed this still happens when you refer to derive-trait by a fully specified name. It's enough that one of the types is in its full form and then none work.

Minimal example (which, to be honest, is a little ridiculous):
```rust
#[derive(Copy, std::marker::Clone)]
struct X;
```
and a slightly more real-world example:
```rust
#[derive(prost::Message, Copy, Clone)]
struct Message {
  payload: u64,
}
```

**Comment by krojew** [comment]
Can confirm it's happening in the latest version. Pretty annoying in simple getters, e.g.

    fn get_time_multiplier(&self) -> f32 {
        self.time_multiplier // Cannot move
    }

**Comment by Undin** [comment]
@krojew I don't think that your case has the same cause that described in issue

**Comment by aloucks** [comment]
I've noticed this with the `glfw` crate's global `FAIL_ON_ERRORS` static variable.


![image](https://user-images.githubusercontent.com/221559/77493663-180e0b00-6e1a-11ea-809d-8c6e12f7894d.png)

The type is `Copy`.

https://github.com/PistonDevelopers/glfw-rs/blob/c51524b52e2e5cd7411b0e7d1cfe3e4eb110d4d6/src/lib.rs#L428

https://github.com/PistonDevelopers/glfw-rs/blob/c51524b52e2e5cd7411b0e7d1cfe3e4eb110d4d6/src/lib.rs#L377

**Comment by Undin** [comment]
@aloucks Your issue is not related to this one. Please, create separate issue
